src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Interface/IPolicyInformationPoint.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Models;
+
+namespace Altinn.Platform.Authorization.Services.Interface
+{
+    /// <summary>
+    /// Defines the interface for the Policy Information Point
+    /// </summary>
+    public interface IPolicyInformationPoint
+    {
+        /// <summary>
+        /// Gets rules
```

Comment:
This comment could hold more information about how the method identifies the rules to return.

Reply:
Added a more detailed summary for GetRulesAsync